spec/models/contact_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Contact, type: :model do
+  describe "associations" do
+    it { should belong_to(:signature) }
+  end
+
+  describe "validations" do
+    it { should validate_presence_of(:signature) }
+    it { should validate_presence_of(:address) }
+    it { should validate_presence_of(:phone_number) }
+    it { should validate_length_of(:address).is_at_most(500) }
+    it { should_not allow_value('6'*32).for(:phone_number).with_message(:too_long) }
```

Comment:
Again, expect > should - also add some air around the `'6'*32`, i.e. `'6' * 32` instead.

Reply:
:sparkles: :sparkles: :sparkles: